ee/models/__init__.py

```diff
@@ -15,4 +16,5 @@
     "Role",
     "RoleMembership",
     "EnterprisePropertyDefinition",
+    "FeatureFlagRoleAccess",
```

Comment:
@liyiy this is the reason when running tests locally you'd get the `ee_ffroleaccess` table doesn't exist, even when it existed in the test db. Wasn't registered in django